.packit.yaml

```diff
@@ -0,0 +1,46 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+specfile_path: rubygem-hammer_cli_foreman_leapp.spec
+
+# add or remove files that should be synced
+files_to_sync:
+    - rubygem-hammer_cli_foreman_leapp.spec
+    - .packit.yaml
+
+# name in upstream package repository or registry (e.g. in PyPI)
+upstream_package_name: hammer_cli_foreman_leapp
+# downstream (Fedora) RPM package name
+downstream_package_name: rubygem-hammer_cli_foreman_leapp
+
+upstream_tag_template: "{version}"
+
+actions:
+  post-upstream-clone:
+    - "wget https://raw.githubusercontent.com/theforeman/foreman-packaging/rpm/develop/packages/plugins/rubygem-hammer_cli_foreman_discovery/rubygem-hammer_cli_foreman_discovery.spec -O rubygem-hammer_cli_foreman_discovery.spec"
+  get-current-version:
+    - ruby -rrubygems -e 'puts Gem::Specification::load(Dir.glob("*.gemspec").first).version'
+  create-archive:
+    - gem build hammer_cli_foreman_leapp.gemspec
+    - bash -c "ls -1t ./hammer_cli_foreman_leapp-*.gem | head -n 1"
+
+jobs:
+  - job: copr_build
+    trigger: pull_request
+    targets:
+      rhel-8:
+        additional_modules: "foreman-devel:el8"
+        additional_repos:
+          - https://yum.theforeman.org/releases/nightly/el8/x86_64/
+          - https://yum.theforeman.org/plugins/nightly/el8/x86_64/
+      rhel-9:
+        additional_modules: "foreman-devel:el9"
+        additional_repos:
+          - https://yum.theforeman.org/releases/nightly/el9/x86_64/
+          - https://yum.theforeman.org/plugins/nightly/el9/x86_64/
+    module_hotfixes: true
+
+srpm_build_deps:
+  - wget
+  - rubygems
+Footer
```

Comment:
```suggestion
```

Hello! Looks like the YAML parser does not like this part.

```
"  parser says\n  in "<unicode string>", line 47, column 1:\n    \n    ^\n  could not find expected \':\' while scanning a simple key"
```

You can always try to validate the config locally, using a `packit validate-config` command (https://packit.dev/docs/cli/validate-config).

If there is anything we can help, [let us know](https://packit.dev/#contact).

František from the Packit team

Reply:
Thank you! :)